### DISCUSSION TITLE
Can I use .bmp images for the custom logo like I could in the old software?

### DISCUSSION THREAD

**Discussion by JosephJilek** [Ralim/IronOS#1091] [Q&A] [answered]
Can I use 96x16 bitmap (.bmp) images for startup in this firmware?

**Comment by Ralim** [comment, marked as answer]
Yes this firmware supports startup images.
However, to keep code size small this firmware does not implement USB at runtime.
Instead you convert your image into a hex file (like the firmware) and then flash it after you have flashed the main firmware.

There are more details here : https://github.com/Ralim/IronOS/blob/master/Documentation/Logo.md

I'm also around to help usually should you get stuck

**Reply by JosephJilek** [reply]
Thanks! I was wondering if it supported .bmp?
